community/event-checklist.md

```diff
@@ -0,0 +1,96 @@
+
+During the core period of the workshop, make sure that they get a **practical hands on experience**(which is what they have come for, otherwise they could have learnt it on YouTube too!)
+
+Make sure to give enough time to cover the queries and concerns of the audience during the workshop.(without queries and interaction, the workshop would not differ from a YouTube Tutorial)
+
+## 2.3 How to organize or approach content:
+
+- **2.3.1 Introduction:** It is a good idea to tell the audience the following things: 
+
+  - What they should take away from the workshop. For example, if it is git workshop, convey that by the end of the workshop they will be comfortable to track their project using git and know how to send a Pull Request properly to a open source repository on Github.
+
+  - Give the audience about how the flow of workshop would be. For example...
+
+    - While hands on, asking them to do them along with the speaker
+    - There will be small doubt sessions periodically **or** may be if the speaker is comfortable, he/she may ask the audience to interrupt in middle and ask the query. It is a good idea to take breaks after each [microcentury](https://susam.in/blog/microcentury/)
+    - It is a good idea to warn the audience if there is a long strech of theoretical discussion for the next few minutes.
+
+  - **2.3.2 Introducing a new topic**: If you are introducing any new topic, it is suggested to try either of the two methods
+
+    - **First principles approach**: Introducing the topic from the ground up. Don't just bring up a word and say, "we are going to learn about this X topic now". Bring the situation to explain why something of this sought is necessary. For example, in git consider the topic "branching". Introducing a problem in the beginning, where git branching is a super-useful solution, this will help the audience grasp the idea better.
+
+    - **Using Analogy**: It is always a useful thing to give a analogy with something which people already know. For  example, in git consider the idea of staging area. The staging area is [generally difficult to grasp](https://github.com/kossiitkgp/mistakes/blob/master/git-workshop.md#git-workshop-2018) for someone new to git. To explain this, one can use "shopping in a mall"  analogy. 
+
+      - Selecting items from the rack <-> Making changes in the files
+
+      - Adding items in the shopping cart <-> Adding files in the staging area using `git add `
+
+      - Billing at the counter <-> Commiting the changes
+
+        The above analogy was inspired from a [stack exchange answer by Spoike](https://softwareengineering.stackexchange.com/questions/119782/what-does-stage-mean-in-git), Later modified by Pankaj Kumar. Good analogies can be found by searching for intuitive explanations for the topic on web and also by brainstorming with the team. The above analogy was a result of both!
```

Comment:
```suggestion
        The above analogy was inspired from a [stack exchange answer by Spoike](https://softwareengineering.stackexchange.com/questions/119782/what-does-stage-mean-in-git), Later modified by @Shankusu993 . Good analogies can be found by searching for intuitive explanations for the topic on web and also by brainstorming with the team. The above analogy was a result of both!
```

Reply:
I was afraid that his username might change. But his original name will never change! Hence I put that
But ya, putting by username is a cool thing

If @Shankusu993 , if u plan to change ur username in future. U can add ur name. 
Currently, accepting the commit suggestion - using username